src/views/affiliations.ts

```diff
@@ -68,6 +69,12 @@ export class AffiliationsView extends BlockView<EditableBlockProps> {
     elements
       .sort(this.sortAffiliations)
       .forEach((e) => this.container.appendChild(e.element))
+
+    const { doi } = this.view.state.doc.attrs
```

Comment:
This looks a bit out of place as this doesn't have much to do with the affiliations. Also, I wonder about a possibility for cases when there no affiliations in this article ass affiliations are an option part of the document. I think a more solid approach would be to create a plugin that would append the DOI html to the affiliations node or if no affiliation node is present, then to whatever is the node above it.

Reply:
As we discussed I created plugin for adding doi into editor....